src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+
+## Verify Setup {#verify-setup}
+
+Verify your setup by intentionally sending an error that breaks your application. Sentry will send an alert to the email address configured on [Sentry.io](http://sentry.io/) notifying you about the event.
+
+An *event* is one instance of sending data to Sentry. Generally, this data is an error. An *issue* is a grouping of similar events.
+
+For example, in <!--SDK name-->, 
+<!--
+Guideline: Add verify setup example for the SDK you are documenting
+-->
+Resolve the created error by either: 
+
+- Clicking `View on Sentry` in the alert sent to the email provided when you established your account
+- Logging in to Sentry, then opening your project
+
+On the **Issue Details** page, scroll down to the Exception stack trace and view the error. 
+
+## Capture Errors {#capture-errors}
+
+The reporting of an *event* - an error, exception, or rejection - is called *capturing*. When an event is captured, it’s sent to Sentry, then creates a new issue group (or is added to an already existing one) based on Sentry’s grouping algorithm. 
+<!--
+Guideline: Add SDK specific information, as appropriate
+-->
+Learn more about how to manually capture errors or enable message capture with the <!--SDK name--> SDK in Capture Errors <!--link to the Capture Errors content for this SDK-->.
+
+### Enrich Error Data {#enrich-error-data}
+
+When the SDK sends an event to Sentry, the event is enriched with data to help identify the source of the event. We include data both pertinent to the event as well as what led up to the event. Much of this data is sent automatically - including the error context <!--link to the Event Context content for this SDK--> and environment <!--link to the Manage Environments--> - as well as the trail of events that happened prior to an issue, which we call breadcrumbs <!--link to the breadcrumbs content for this SDK-->. You don't need to configure these, though you may modify them. 
+
+Learn more about enriching data sent with events in Enrich Event Data <!--link to the Enrich Event Data page for this SDK-->.
+
+### Set the Release Version {#set-release-version}
+
+When you configure Sentry to include the version of your application, Sentry can tell you about regressions as well as detailed information about the suspect commit. 
+
+Use the <!--SDK setting-->:
+
+<!-- SDK configuration example -->
+
+After you inform Sentry of a new release, you will see information about it, such as new issues and regressions introduced in the release.
+
+Learn more about what releases can do, including using a repository integration, creating the release and associated commits, and telling Sentry when you deploy a release in Track Releases. <!--link to the Track Releases page for this SDK-->.
+
+### <SDK-Specific Setup>
+
+Here, add information that is particular to the SDK you are documenting. For example, in the JavaScript SDK, this is where we discuss setting up Source Maps.
+
+## Monitor Performance {#monitor-performance}
```

Comment:
@benvinegar should look at this

Reply:
This section is directly from the Notion doc Ben and I are working on for a Performance Monitoring overview page (I've been moving the content from that doc into the SDK redesign doc). But I welcome Ben's input here, too.